src/hpack.app.src

```diff
@@ -6,9 +6,12 @@
   {registered, []},
   {applications, [
                   kernel,
-                  stdlib,
-                  lager
+                  stdlib
                  ]},
-  {env, []}
+  {env, []},
+
+  {maintainers, ["Joe Devivo"]},
```

Comment:
I think it's "Joe DeVivo" (according to [GitHub](https://github.com/joedevivo)) :)